solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Home/HomeView.swift

```diff
@@ -43,27 +50,49 @@ final class HomeView: UIView {
         self.restaurants = restaurants
         self.tableView.reloadData()
     }
+    
+    func updateLoading(with isLoading: Bool) {
+        loadingView.isHidden = !isLoading
+        if isLoading {
+            loadingView.updateLoading(isLoading)
+            loadingView.setLoadingMessage("Carregando Restaurantes...")
+        } else {
+            loadingView.updateLoading(isLoading)
+        }
+    }
```

Comment:
Uma vez que você adicionou a lógica de startAnimating dentro do LoadingView, você não precisa mais fazer esse if aqui.
```suggestion
    func updateLoading(with isLoading: Bool) {
        loadingView.isHidden = !isLoading
        loadingView.updateLoading(isLoading)
    }
```